server/llm_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -227,7 +227,7 @@ async def create_text_generation_inference_bundle(
                     schema_location="TBA",
                     flavor=StreamingEnhancedRunnableImageFlavor(
                         flavor=ModelBundleFlavorType.STREAMING_ENHANCED_RUNNABLE_IMAGE,
-                        repository="text-generation-inference",  # TODO: let user choose repo
+                        repository="ghcr.io/huggingface/text-generation-inference",  # TODO: let user choose repo
```

Comment:
@yunfeng-scale It turns out I need to update TGI repo name in order to skip image existence check in ECR repo given the logic here https://github.com/scaleapi/llm-engine/blob/1370d791a0056194617665fa6c76afebd5ae677b/server/llm_engine_server/domain/use_cases/model_bundle_use_cases.py#L386

Is this change reasonable? Should we back propagate this back to hmi as well?

Reply:
we shouldn't hardcode this since it diverged internal / OSS code, can you add this as a parameter?